packages/luna/src/components/Lessons/LessonCard.tsx

```diff
@@ -11,39 +11,51 @@ import { Menu, type MenuAction } from "@/components/Menu";
 import CalendarEdit from "@litespace/assets/CalendarEdit";
 import CalendarRemove from "@litespace/assets/CalendarRemove";
 import CheckCircle from "@litespace/assets/CheckCircle";
-import { Tooltip } from "@/components/Tooltip";
 
 export type Props = {
   start: string;
   duration: number;
   canceled: "tutor" | "student" | null;
-  tutor: {
+  member: {
     id: number;
     name: string | null;
     image: string | undefined;
+    // NOTE: role shall be student when the cur user is tutor and vice versa
```

Comment:
Here how can we improve this: 
1. Using JS Docs will allow you comment to be visible for the other team members without the need to open the component source code directlly. 
2. Fix the typos (use complete words). 

```suggestion
    /**
    * @note role shall be `student` when the current user is tutor and vice versa
    */
```